dump-persistence-schema-compiler-plugin/src/test/resources/DataEnum.scala

```diff
@@ -8,7 +8,7 @@ object DataEnum {
   case object Two extends Enum
   case object Three extends Enum
 
-  case class Data(enum: Enum)
+  case class Data(`enum`: Enum)
```

Comment:
Just curious, why is this required?

Reply:
It will become a keyword in Scala 3 (enum) and from Scala 2.13.7 there's a deprecation warning and because of that one test returns that as a result. 
```
"[warning: 1 deprecation (since 2.13.7); re-run with -deprecation for details [info]]" 
was not equal to "[]"
```